spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_project do
+      status { 'done' }
```

Comment:
Symbol型として記載しましょう。Enumの値であることを明示できます。
```suggestion
      status { :done }
```

Reply:
b7afa49で修正しました。